### PR TITLE
add about page

### PR DESCRIPTION
**Pull request by tania833**

### CODE REVIEW

File: src/pages/About/components/Barvy/Barvy.jsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+Barvy.propTypes = {
+  barvyProps: {
+    src: PropTypes.string,
+    alt: PropTypes.string,
+    para1: PropTypes.string,
+    para2: PropTypes.string,
+    para3: PropTypes.string,
+  },
+};
+
```
Comment (ostannja) [inline src/pages/About/components/Barvy/Barvy.jsx:35]: great job

File: src/pages/About/components/Ship/Ship.jsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+            однодумців. Це однокурсники, викладачі та команда Барвів. Разом ми
+            створюємо особливу атмосферу всередині нашого онлайн-простору.{' '}
+          </p>
+        </div>
+      </div>
+    </div>
+  );
+};
+
+export default Ship;
```
Comment (ostannja) [inline src/pages/About/components/Ship/Ship.jsx:30]: empty line

File: src/App.css [hunk 3]
```diff
@@ -93,6 +93,4 @@ a {
   html {
     font-size: 8px
   }
-}
-
-
+}
```
Comment (ostannja) [inline src/App.css:96]: empty line

File: src/pages/About/components/Ship/Ship.module.scss [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+}
+
+    .photoBlock{
+        display: none;
+    }
+    
+    .textBlockPara{
+        font-size: 1.8rem;
+    }
+  }
```
Comment (ostannja) [inline src/pages/About/components/Ship/Ship.module.scss:103]: empty line

File: src/pages/About/components/Start/Start.module.scss [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+
+    .workingPictureContainer, .photographingPictureContainer {
+        margin: 2rem auto;
+    }
+
+    .workingPara, .photographingPara {
+        font-size: 1.8rem;
+        max-width: 25rem;
+    }
+}
```
Comment (ostannja) [inline src/pages/About/components/Start/Start.module.scss:98]: empty line

File: src/pages/About/components/Title/TitleBarvy.module.scss [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+
+  @media all and (max-width: 576px) {
+    .subContainer{
+        width: 30rem;
+    }
+    
+    .titleContainer{
+        margin-top: -5rem;
+    }
+}
```
Comment (ostannja) [inline src/pages/About/components/Title/TitleBarvy.module.scss:25]: empty line

File: src/pages/About/components/Ship/Ship.module.scss [hunk 7]
```diff
@@ -0,0 +1,86 @@
...
+    justify-content: space-between;
+    width: 100vw;
+    height: 30.9375rem;
+    background-color: #E7F4FA;
+}
+
+.photoBlock{
+    margin-top: -9.6875rem; 
+    height: 40.625rem;
+
```
Comment (SvitlanaKalmutska) [inline src/pages/About/components/Ship/Ship.module.scss]: Empty line

File: src/pages/About/components/Barvy/Barvy.module.scss [hunk 8]
```diff
@@ -0,0 +1,86 @@
...
+  }
+.textBlock__para{
+  width: 35rem;
+}
+  .textBlock{
+    margin: 0 auto;
+  }
+}
+
+
```
Comment (SvitlanaKalmutska) [inline src/pages/About/components/Barvy/Barvy.module.scss]: two empty lines

File: src/pages/About/components/Barvy/Barvy.module.scss [hunk 9]
```diff
@@ -0,0 +1,86 @@
...
+  }
+}
+
+.imageContainer{
+    height: 21rem;
+    width: 30rem;
+    overflow: hidden;
+    object-fit: contain;
+  @include flex(center, flex-end, row, nowrap);
+
```
Comment (SvitlanaKalmutska) [inline src/pages/About/components/Barvy/Barvy.module.scss:56]: empty line

File: src/pages/About/components/Barvy/Barvy.module.scss [hunk 10]
```diff
@@ -0,0 +1,86 @@
...
+    margin-right: 4rem;
+    font-family: "Mak";
+    font-size: 2.5rem;
+    font-weight: 300;
+    line-height: 2.2rem;
+    box-sizing: border-box;
+    text-align: left;
+    color: #000000;
+    text-transform: uppercase;
+
```
Comment (SvitlanaKalmutska) [inline src/pages/About/components/Barvy/Barvy.module.scss:34]: empty line

**Review by SvitlanaKalmutska — COMMENTED** [review]
Please, look over the code: there are some empty lines inside of it. Except that everything is excellent!

**Pull request merged**